### PR TITLE
chore(v3): backport latest v4 changes

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Description

Backport latest v4 changes.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jfagoagas** [comment]
This PR requires to adapt all the providers thing in v4 to the `audit_info` in v3.

File: docs/developer-guide/unit-testing.md [hunk 1]
```diff
@@ -509,7 +509,113 @@ class Test_compute_firewall_rdp_access_from_the_internet_allowed:
...
+        with patch(
+            "prowler.providers.gcp.lib.service.service.GCPService.__is_api_active__",
+            new=mock_is_api_active,
+        ), patch(
+            "prowler.providers.gcp.lib.service.service.GCPService.__generate_client__",
+            new=mock_api_client,
+        ):
+            # Instantiate an object of class with the mocked provider
+            bigquery_client = BigQuery(
+                set_mocked_gcp_provider(project_ids=[GCP_PROJECT_ID])
```
Comment (jfagoagas) [inline docs/developer-guide/unit-testing.md]: This needs to be `audit_info`, please review the docs pulled from v4 branch.

File: prowler/lib/outputs/summary_table.py [hunk 2]
```diff
@@ -40,11 +40,13 @@ def display_summary_table(
             entity_type = "Project ID/s"
             audited_entities = ", ".join(audit_info.project_ids)
 
-        if findings:
+        # Check if there are findings and that they are not all MANUAL
+        if findings and not all(finding.status == "MANUAL" for finding in findings):
```
Comment (jfagoagas) [inline prowler/lib/outputs/summary_table.py]: Is manual present in v4?

File: prowler/lib/outputs/summary_table.py [hunk 3]
```diff
@@ -68,16 +70,17 @@ def display_summary_table(
                 ):
                     add_service_to_table(findings_table, current)
 
-                    current["Total"] = current["Critical"] = current["High"] = current[
-                        "Medium"
-                    ] = current["Low"] = 0
+                    current["Total"] = current["Pass"] = current["Muted"] = current[
```
Comment (jfagoagas) [inline prowler/lib/outputs/summary_table.py]: `muted` is not present in v3.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/lib/outputs/summary_table.py (hunk 3 above)
Comment (MrCloudSec) [reply prowler/lib/outputs/summary_table.py]: Done!

File: prowler/lib/outputs/summary_table.py (hunk 2 above)
Comment (MrCloudSec) [reply prowler/lib/outputs/summary_table.py]: Fixed!

File: docs/developer-guide/unit-testing.md (hunk 1 above)
Comment (MrCloudSec) [reply docs/developer-guide/unit-testing.md]: Fixed!

**Review by jfagoagas — APPROVED** [review]
Great job @sergargar 👏

**Pull request merged**